### PR TITLE
add pytz

### PR DESCRIPTION
**Pull request by msarahan**

### CODE REVIEW

File: recipes/pytz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pytz
+
+about:
+  home: http://pytz.sourceforge.net/
```
Comment (jakirkham) [inline recipes/pytz/meta.yaml]: They list ( http://pythonhosted.org/pytz/ ) on [PyPI]( https://pypi.python.org/pypi/pytz ).

File: recipes/pytz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  license: MIT
+  license_file: LICENSE.txt
+  summary: World timezone definitions, modern and historical
+  description: |
+    pytz brings the Olson tz database into Python. This library allows
+    accurate and cross platform timezone calculations using Python 2.4 or
+    higher. It also solves the issue of ambiguous times at the end of daylight
+    saving time.
+  doc_url: http://pytz.sourceforge.net/
```
Comment (jakirkham) [inline recipes/pytz/meta.yaml]: Same comment as with home.

**Comment by jakirkham** [comment]
Some very minor errata in the metadata. Otherwise LGTM (given CI passes).

**Comment by msarahan** [comment]
Thanks for checking those URLs.

**Comment by jakirkham** [comment]
Of course.

**Comment by jakirkham** [comment]
Will let this one sit for a bit as well to see if anyone else has feedback.

**Comment by jakirkham** [comment]
Thanks @msarahan.

**Pull request merged**